src/diagnostics/define_diagnostics.jl

```diff
@@ -720,7 +720,7 @@ function define_diagnostics!(land_model)
         short_name = "scms",
         long_name = "Soil CO2 Microbial Source",
         standard_name = "soil_co2_microbial_source",
-        units = "kg C m^-2 s^-1",
+        units = "mol CO2 m^-2 s^-1",
```

Comment:
I think this is kg C still, right?

Reply:
correct - and actually kg C m^-3 (volume) - thanks!